python_ta/reporters/core.py

```diff
@@ -91,23 +94,28 @@ def has_messages(self) -> bool:
         """Return whether there are any messages registered."""
         return any(messages for messages in self.messages.values())
 
-    def set_output(self, filepath: Optional[str] = None) -> None:
+    def set_output(self, filepath: Optional[Union[str, IO]] = None) -> None:
```

Comment:
Oh I forgot to mention, please rename this parameter to `out` (since it's no longer a path)